smartMeter/ngsild-payloads/smarMeter.json

```diff
@@ -0,0 +1,64 @@
+{
+    "id": "urn:ngsi-ld:Device:smartMeter01",
+    "type": "Device",
+    "name": {
+        "type": "Property",
+        "value": "smartMeter0121215"
+    },
+    "serialNumber": {
+        "type": "Property",
+        "value": "X214356"
+    },
+    "dateInstalled": {
+        "type": "Property",
+        "value": "2020-03-11T11:00:00Z"
+    },
+    "dateLastCalibration": {
+        "type": "Property",
+        "value": "2020-06-11T11:00:00Z"
+    },
+    "controlledProperty": {
+        "type": "Property",
+        "value": [
+            "waterConsumption",
+            "minFlow",
+            "maxFlow",
+            "persistenceFlowDuration"
+        ]
+    },
+    "alarm": {
```

Comment:
controlledProperty / alarm is in both model and smartMeter

Reply:
Aalarms are deleted. The controlledProperty is putted actually in both models follwing the Fiware recommendations.